src/app/tests/TestReadInteraction.cpp

```diff
@@ -1946,7 +1947,12 @@ void TestReadInteraction::TestSubscribeWildcard(nlTestSuite * apSuite, void * ap
         // items for the other.
         //
         // Thus we should receive 29*2 + 4 + 3 = 65 attribute data in total.
-        NL_TEST_ASSERT(apSuite, delegate.mNumAttributeResponse == 65);
+        constexpr size_t kExpectedAttributeResponse = 65;
+#else
+        // Receiving 29*2 + 4 + 6 = 68 when eventlist attribute is not available.
```

Comment:
Please clearly explain where that "+ 4 + 6" comes from.  Because I don't see where "+6" would come from.  Are we getting 1 + 5 for both of the chunked cases because the packet boundaries shifted (so this should be "+ 5 + 5" with an appropriate comment)?  Or is something else going on?

Reply:
I got this from a previous historical comment ... explanation of how things are separated did not seem as clear to me as to update.

Reply:
I wrote the "historical comment", so apologies for it not being as clear as it perhaps should have been.  The upshot is that we should have 58 responses if no lists got chunked.  If lists got chunked, we should get one extra response for every item in the list that did not fit in the initial list-start packet.  The lists have length 6 to start with, so somewhere between 0 and 6 items will fit in that first packet.

Reply:
Updated the comment a bit to try to guide the future reader of the code, however actual chunking is still unclear to me. We used to have 68 here in previous PRs, but how that turned into 65 based on event list is absolutely unclear to me.

Reply:
Would need some comments regarding global attributes, however those do not seem discussed in the comments here. I would have expected global attributes for each cluster (so 3 * <something? 4?>) however none of those actually show up right now in the existing comments. However removing a global element would explain a delta of 3 ... but in the other direction :/